pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_creating-a-product.adoc

```diff
@@ -1,37 +1,36 @@
 
 .Prerequisites
 
-* You are logged in to {ProductShortName}.
-* You are assigned the publisher role.
+* You are logged in.
+* You have Publisher permissions.
 
 .Procedure
 
 . Navigate to menu:Products[New Product].
-
-. Complete the following information:
-  Product Name:: Enter the name for your product. The product name is displayed in {ProductShortName}, the document preview, and in user-facing content.
-  Product UrlFragment:: Enter the product URL fragment. The URL fragment is displayed in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
-  Product Description (optional):: Provide a description of the product.
-
+. Enter the following information, items marked with an asterisk (*) are mandatory:
+[horizontal]
+  Product Name *:: The text that appears in {ProductShortName}, the document preview, and in user-facing content.
+  Product URL Fragment *:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
+  Product Description:: A description of the product.
+  Product Version *:: The product version. This value is replaced by the value of the product version or version number attribute in user-facing content that {ProductShortName} generates.
+  Version URL Fragment *:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
 . Click btn:[Save].
 
-After creating a product, you need to assign a product version.
-
-.Example {ProductShortName} product configuration
-
+.Example
+[horizontal]
 Product Name:: .NET Core
-Product UrlFragment:: `net_core`
+Product URL Fragment:: net_core
 Product Description:: Documentation for the .NET Core framework.
+Product Version:: 4.4-Beta
+Version URL Fragment:: 4.4-beta
 
-Entering the above information creates a {ProductShortName} product named `.NET Core`. The URL for documentation that is associated with this product includes `net_core` to identify the product. For example, after creating the product version `4.4-beta` and assigning a module the `*.NET Core*` product and version `*4.4-beta*`, the resulting URL for that module is:
+Entering the above information creates a {ProductShortName} product named *.NET Core*. The URL for documentation that is associated with this product includes the URL Fragment *net_core* to identify the product, and the Version URL Fragment is *4.4-beta*.
 
-`\https://access.stage.redhat.com/documentation/en-us/topic/net_core/4.4-beta/_module-UUID_`
+The resulting URL for that module is:
 
-// .Additional resources
-// link:placeholder.url.com[Creating a product version]
-// link:placeholder.url.com[{ProductName} user roles]
+`\https://access.stage.redhat.com/documentation/en-us/topic/net_core/4.4-beta/_module-UUID_`
```

Comment:
This example is good, if we are considering our hostname as https://access.stage.redhat.com,  it could be better if it is more generic.
Also, It will be good to have a Guide URL construction similarly.